code/modules/media/jukebox.dm

```diff
@@ -777,6 +778,13 @@ var/global/list/loopModeNames=list(
 		"upbeathypedancejam" = "Dance"
 	)
 
+/obj/machinery/media/jukebox/bar/New()
+	..()
+	var/MM = text2num(time2text(world.timeofday, "MM"))
+	if(MM == 10)
+		playlists.Add("halloween" = "Halloween")
```

Comment:
`Runtime in jukebox.dm, line 785: illegal use of list2args() or named parameters`

Reply:
Good catch.  Sorry for merging this without seeing it myself.

Reply:
how should I fix this properly, sensei? @DamianX

Reply:
`playlists["halloween"] = "Halloween"`